CONTRIBUTING.md

```diff
@@ -14,12 +14,12 @@ All Contributions should be contained with a folder that describes the content o
   * Known gotchas, system requirements, compatibility issue, etc.
   * Logstash config files to process and ingest data
   * Template with index mappings (if needed)
-  * Kibana config file to load a prebuilt Kibana dashboard  
+  * Kibana config file to load a prebuilt Kibana dashboard
   * Any other code that is a part of the instruction set. Scripts should be documented and dependencies available in a standard format e.g.for Python a requirements.txt and pip would be appropriate. Always detail tested language versions for scripts e.g.Python 3.5.x
 
 * **Data** <br>
-  You can either provide the data file with the example (for small sample datasets), or provide instructions / link to download the raw data (or Elasticsearch index snapshot) from another source (such as Amazon S3). 10mb is a reasonable threshold before moving to an external download- but this shouldn't be considered a hardline.
-  
+  You can either provide the data file with the example (for small sample datasets), or provide instructions / link to download the raw data (or Elasticsearch index snapshot) from another source (such as Amazon S3). 10 MiB is a reasonable threshold before moving to an external download- but this shouldn't be considered a hardline.
```

Comment:
Should be 10MB?

Reply:
Not really. MB is ambiguous. MiB is better defined which is why you will find good technical documentation often uses it
https://en.wikipedia.org/wiki/Byte#Multiple-byte_units.